stories/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I think for clarity between these two `tsconfig` files and their differences it might be clearer to add a third file, `sharedTsconfig.json`, `tsconfigShared`, or `tsconfigBase`, and pull them into both `stories/tsconfig.json` and `tsconfig` at the root. That way we can see what properties apply specifically to the root and what apply specifically to the stories. What do you think?

Reply:
done. Yes it is much simpler now, I also removed some configs that were already in the default values, except for `jsx` for some reason it doesnt work without putting it with default